### PR TITLE
[16.0][FIX] pos_partner_birthdate: fixing this.changes state

### PR DESCRIPTION
**Pull request by imbarbudiman**
use `useState` when add `birthdate_date` to `changes` state

### CODE REVIEW

File: pos_partner_birthdate/static/src/js/ClientDetailsEdit.esm.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 /** @odoo-module **/
+import {useState} from "@odoo/owl";
```
Comment (chienandalu) [inline pos_partner_birthdate/static/src/js/ClientDetailsEdit.esm.js]: You've got `owl` as a global object
```suggestion
const {useState} = owl;
```

**Review by chienandalu — COMMENTED** [review]
LG @imbarbudiman :+1: 

please merge your commits into one

nitpick: a better desprition of the commit: *[FIX] pos_partner_birthdate: PartnerDetailsEdit reactivity*

**Comment by imbarbudiman** [comment]
> LG @imbarbudiman 👍
> 
> please merge your commits into one
> 
> nitpick: a better desprition of the commit: _[FIX] pos_partner_birthdate: PartnerDetailsEdit reactivity_

commits merged

**Review by chienandalu — APPROVED** [review]

**Review by ecino — APPROVED** [review]
LGTM :+1:

**Pull request merged**